webdriver/tests/classic/perform_actions/wheel.py

```diff
@@ -55,15 +56,12 @@ def test_scroll_iframe(session, test_actions_scroll_page, wheel_chain):
 
     wheel_chain.scroll(0, 0, 5, 10, origin=target).perform()
 
-    events = get_events(session)
+    # Chrome requires some time (~10-20ms) to process the event from the iframe, so we wait for it.
+    def wait_for_events(_):
+        return len(get_events(session)) > 0
 
-    timeout_start = time.time()
-    # Chrome requires some time (~10ms) to process the event from the iframe, so try to get events until they are in
-    # place, for not longer than for 500ms.
-    while time.time() < timeout_start + 0.5:
-        events = get_events(session)
-        if len(events) > 0:
-            break
+    Poll(session, timeout=0.5, interval=0.01, message='No events found').until(wait_for_events)
```

Comment:
Can we make that identical to the BiDi case please? Also lets add `wheel` to the message so that it's clearer what we are waiting for. I'm fine with specifying a smaller delay to speed up the processing which we would also need for the BiDi case.

Reply:
Agree, done